vitest.config.mts

```diff
@@ -3,7 +3,7 @@ import { defineConfig } from 'vite';
 
 export default defineConfig({
   test: {
-    root: 'src',
+    include: ['src/**/*.spec.ts'],
```

Comment:
I think just doing `dir: 'src'` would be more succinct and wouldn't rewrite the default for `include`, but either works.. [`dir`](https://vitest.dev/config/#dir) changes the directory without changing the default for [`include`](https://vitest.dev/config/#include), which is: `['**/*.{test,spec}.?(c|m)[jt]s?(x)']`; now `.test.tsx` files wouldn't work for example. Though it doesn't really matter in this case.
```suggestion
    dir: 'src',
```